README.md

```diff
@@ -6,3 +6,145 @@
 This repository contains mock server that is used for advanced E2E tests for our proxy integrations.
 When the integration is properly configured, it allows us to inspect requests sent from these integrations and perform necessary assertions.
 
+
+## Preperation
+
+In order to test your proxy integration, you need to configure it to send requests to this mock app.
+In most cases, this can be done by setting these two ENV variables when building the integration: `FPCDN` and `INGRESS_API`. For example:
+
+```bash
+FPCDN=<API_URL> INGRESS_API=<API_URL> pnpm build
+```
+
+Refer to specific integration README for more details.
+
+> **Note**
+> In order to get the API URL, contact the DX team.
+
+## How to use it
+
+This app exposes REST API that can be used to run the tests.
+
+To run tests, send `POST` request to: `/api/test/run-tests` with following payload:
+```json
+{
+  // Your proxy host
+  "host": "https://mock-test-inter-568-mock-app-tests.cfi-fingerprint.com",
+  // CDN endpoint in your proxy integration
+  "cdnProxyPath": "worker/pxdownload",
+  // Ingress endpoint in your proxy integration
+  "ingressProxyPath": "worker/pxresult",
+  // First part of ii parameter sent by our proxy integration: <trafficName>/<integrationVersion>/type
+  "trafficName": "fingerprint-pro-akamai",
+  // Second part of ii parameter sent by our proxy integration: <trafficName>/<integrationVersion>/type
+  "integrationVersion": "1.0.1-snapshot.0"
+}
+```
+
+You will receive following response:
+```json
+{
+  "host": "https://mock-test-inter-568-mock-app-tests.cfi-fingerprint.com",
+  "status": "finished",
+  "results": [
+    {
+      // True if the test passed
+      "passed": true,
+      // Name of the test
+      "testName": "agent request query params",
+      // Duration of the request in milliseconds
+      "requestDurationMs": 322
+    },
+    {
+      // False if the test failed
+      "passed": false,
+      // Name of the test
+      "testName": "ingress request headers",
+      // Duration of the request in milliseconds
+      "requestDurationMs": 23,
+      // Reason why the test failed
+      "reason": "Expected header 'X-Forwarded-For' to be undefined",
```

Comment:
Reason changed for explaining which field is throw the error, please see the screenshot below
<img width="590" alt="image" src="https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/assets/5910168/6114be6e-1971-48d4-b8b4-ca16e427e8a1">

Reply:
Done!